src/components/Table/TablePropTypes.js

```diff
@@ -198,6 +200,8 @@ export const defaultI18NPropTypes = {
   filterNone: 'Unsort rows by this header',
   filterAscending: 'Sort rows by this header in ascending order',
   filterDescending: 'Sort rows by this header in descending order',
+  rowCountLabel: 'Results',
```

Comment:
I think you can remove `rowCountLabel` here, as well as the other locations. It's not being used now that `rowCountInHeader` takes a param of `rowCountLabel`

Reply:
@tay1orjones rowCountLabel: 'Results' is removed from TablePropTypes.js but same is using in Table.jsx because it's default value is passing in function  rowCountHeaderField: i18n.rowCountInHeader(
            view.pagination.totalItems,
            i18n.rowCountLabel
          ),
If we are not passing label value from storybook then it will set the from default props and I tested the same. 
Please review the same.